spring-boot/proxy/src/main/java/eu/coatrack/proxy/metrics/MetricsCounterService.java

```diff
@@ -50,12 +52,18 @@ public class MetricsCounterService {
     public MetricsCounterService(MeterRegistry meterRegistry, MetricsTransmitter metricsTransmitter) {
         this.meterRegistry = meterRegistry;
         this.metricsTransmitter = metricsTransmitter;
+        // Quick fix of #237 by setting default time zones of Admin and Gateway to CET timezone.
+        // In the future, this system should be overhauled by the introduction of ZonedDateTime's.
+        TimeZone.setDefault(TimeZone.getTimeZone(ZoneId.of("Europe/Berlin")));
     }
 
     public void increment(TemporaryMetricsAggregation tma) {
         logBeginningOfIncrementation(tma);
-        int currentCount = incrementCounterAndReturnCurrentValue(tma);
-        Metric metricToTransmit = createMetricToTransmit(tma);
+
+        ZonedDateTime cetTimeNow = ZonedDateTime.now(ZoneId.of("Europe/Berlin"));
```

Comment:
As the JVM time was already set to `CET`, it seems there is no need to set the timezone again here?!

Besides, as neither gateway nor admin actually use the time information, I assume that `LocalDate.now()` should be sufficient here.

Reply:
You are right. `LocalDate now = LocalDate.now()` is sufficient here. However, transforming it to a Date, the Timezone has to be provided, too:
```
java.util.Date.from(now.atStartOfDay().toInstant(ZoneOffset.UTC))
```